torBot.py

```diff
@@ -88,18 +88,34 @@ def get_all_emails(soup):
             if 'http' in email_link:
                 websites.append(email_link)
             elif 'mailto' in email_link:
-                emails.append(email_link)
+                """Split email address on"""
+                email_addr = email_link.split(':')
+                emails.append(email_addr[1])
         else:
             pass
-    return websites,emails
-        
+    if pretty_print == False:
+        return websites,emails
+    elif pretty_print == True:
+        """Pretty print output as below"""
+        print ('') 
+        print ('Mails Found - '+str(len(emails)))
+        print ('-------------------------------')
+        for mail in emails:
+            print (mail)
+        print ('') 
+        print ('') 
+        print ('Websites Found - '+str(len(websites)))
+        print ('-------------------------------')
+        for web in websites:
+            print (web)
+        return ''       
 
 def main():
  header()
  stemTest()
  print ("Tor Ip Address :")
  a = readPage("http://www.whatsmyip.net/")
- print (get_all_emails(a))
+ print (get_all_emails(a, pretty_print=True))
```

Comment:
Use argument parser would be better.
for example` torBot.py` for normal printing and `torBot.py -P` for pretty print